src/Native/System.Native/pal_networking.cpp

```diff
@@ -2013,3 +2018,21 @@ extern "C" Error WaitForSocketEvents(int32_t port, SocketEvent* buffer, int32_t*
 
     return WaitForSocketEventsInner(port, buffer, count);
 }
+
+extern "C" int32_t PlatformSupportsMultipleConnectAttempts()
+{
+#if defined(PLATFORM_SUPPORTS_MULTIPLE_CONNECT_ATTEMPTS)
```

Comment:
IIRC clang will emit a warning in the first case if the macro is not defined. I could easily be wrong here, though.